test/Image_Tests/src/Codecs_Spec.enso

```diff
@@ -0,0 +1,38 @@
+from Standard.Base import all
+import Standard.Base.System.Process
+import Standard.Base.System.Process.Exit_Code
+import Standard.Test
+
+import Standard.Image.Codecs
+
+polyglot java import java.lang.System as Java_System
+
+fetch addr file =
+    case Process.run_command "curl" [addr, "--silent", "--output", file.path] of
+        Exit_Code.Exit_Success ->
+            file
+        Exit_Code.Exit_Failure code ->
+            Test.fail ("Failed to fetch test image. Exit code: " + code.to_text)
```

Comment:
Tests need to be able to be run without network, and shouldn't fail just because the resource wasn't able to be fetched. A more nuanced approach is needed here.